components/Payment/interfaces/fee_calculator_interfaces.rst

```diff
@@ -0,0 +1,26 @@
+Fee Calculator Interfaces
+=========================
+
+Fee Calculator Interface
```

Comment:
I think we should title chapters about interfaces/classes with their real name, which means: ``Fee Calculator Interface`` => ``FeeCalculatorInterface``. It looks quite strange as it is now, with separated words but still capitalized.
This comment refers to all titles in this PR ;)